test/e2e/cluster_status_test.go

```diff
@@ -59,11 +59,12 @@ func TestBackupStatus(t *testing.T) {
 		if bs == nil {
 			return false, nil
 		}
-		if bs.Backups != 1 {
-			return true, fmt.Errorf("backups = %d, want 1", bs.Backups)
+		// We expect it will make one backup eventually.
+		if bs.Backups < 1 {
+			return false, nil
 		}
 		if bs.BackupSize == 0 {
-			return true, fmt.Errorf("backupsize = 0, want > 0")
+			return false, fmt.Errorf("backupsize = 0, want > 0")
```

Comment:
Internally, it doesn't matter because it checks error first and return immediately.
But there is a hidden rule that events that didn't succeed should return `false` -- Only successful events can return `true`.